bin/cfg/Tomb1Main.json5

```diff
@@ -17,6 +17,10 @@
     // Removes all shotgun and shotgun shells pickups.
     "disable_shotgun": false,
 
+    // Enables showing maximum pickup count and kill count on each level.
+    // This includes unobtainable items.
+    "enable_detailed_stats": true,
```

Comment:
Everything looks good except I would move this setting next to `enable_compass_stats`.